pkg/backend/openshiftcluster.go

```diff
@@ -38,22 +38,36 @@ func (ocb *openShiftClusterBackend) try(ctx context.Context) (bool, error) {
 
 	log.Print("dequeued")
 	atomic.AddInt32(&ocb.workers, 1)
+	ocb.m.EmitGauge("backend.workers.count.openshiftcluster", 1, map[string]string{})
```

Comment:
`ocb.m.EmitGauge("backend.openshiftcluster.workers.count", int64(atomic.LoadInt32(&ocb.workers)), nil)` - then you can omit the -1 later

metrics naming should be `backend\.{openshiftcluster,subscriptions}(\..*)?\.{count,duration}` - i.e. check the ordering

Reply:
also I wouldn't mind having `durationMs` instead of `duration` - wdyt?

Reply:
it will get aggregated to durationMs_Rate, durationMs_Max, etc by default. So this does not look very nice